rplugin/python3/deoplete/sources/alchemist.py

```diff
@@ -19,29 +18,28 @@ def __init__(self, vim):
         self.re_suggestions = re.compile(r'kind:(?P<kind>.*), word:(?P<word>.*), abbr:(?P<abbr>.*), menu:(?P<menu>.*), info:(?P<info>.*)$')
```

Comment:
Same as in neovim-completion-manager script. Using `[^,]` should perform better.

Reply:
@hauleth good catch! fixed